go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/Netflix/go-env
 
-go 1.22
+go 1.23.2
```

Comment:
Is the `.2` minor revision a hard requirement here? If not, let's just make this `1.23` to avoid things like #26

Reply:
Not necessary - updated to just go 1.23 and updated title/description to match ✅